### ISSUE TITLE
Sizes on Unisex Logomark and Logotype tees are in wrong order

### ISSUE THREAD

**Issue by ekecso** [elementary/store#148]
## What Happened

Sizes are displaying in wrong order for logomark and logotype tees

## Expected Behavior

Sizes should display in order: XS, S, M, L, XL, 2XL, 3XL, 4XL

**Comment by Xristodoulos99** [comment]
Can i work on it?

**Comment by cassidyjames** [comment]
@Xristodoulos99 absolutely! Note that the sizes come from the Printful API, so we'll need a way to re-order them before they're displayed.
